src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -899,9 +899,13 @@
     <value>All</value>
     <comment>An option to choose from for the "bell style" setting. When selected, a combination of the other bell styles is used to notify the user.</comment>
   </data>
-  <data name="Profile_BellStyleVisual.Content" xml:space="preserve">
-    <value>Visual (flash taskbar)</value>
-    <comment>An option to choose from for the "bell style" setting. When selected, a visual notification is used to notify the user. In this case, the taskbar is flashed.</comment>

```

Comment:
So, this is annoying. It took me forever to understand what Dustin was telling me about the resw.

All versions of WT have _one_ resw file handed to them. So, if we took this resw file and put it on an older version, we'd crash because `Profile_BellStyleVisual.Content` is missing.

Unfortunately, because of that, you can't actually remove keys from the resw. So we'll just have to keep this one around forever even though we won't be using it in future releases. :(